.github/workflows/scheduled.yml

```diff
@@ -12,14 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: "Scheduled Fuzzer Jobs"
+name: "Fuzzer Jobs"
 
 on:
   pull_request:
-    paths:
```

Comment:
We now want it to run on every PR and not only to test the workflow, but it would make sense to add the path filter we use for the other builds here as well :)